src/app/static/src/tests/components/plots/choropleth/choropleth.test.ts

```diff
@@ -388,8 +389,12 @@ describe("Choropleth component", () => {
             fitBounds: mockMapFitBounds
         };
 
-        const button = wrapper.find(LControl).find('button')
-        expectTranslated(button, 'Reset view', 'Réinitialiser la vue', store);
+        const button = wrapper.find(LControl).find('div').find('a')
+        expect(button.attributes("aria-label")).toBe("Reset view");
```

Comment:
You should be able to do this with `expectTranslated` - it optionally takes an attribute name param if you want to check an attribute rather than `text()`